x/shared/session.go

```diff
@@ -5,6 +5,10 @@ import (
 	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
 )
 
+// closeBlockOffset is the offset to subtract from the earliest supplier commit
+// height to ensure that the supplier has enough time to commit the claim/proof.
+const closeBlockOffset = 1
```

Comment:
The PR description did not make it clear that you're introducing a NEW parameter. 

Haven't thought through everything yet - but the fact that we have like 4 params + grace periods and now we need to add more offsets on top of offsets seems like a code smell.

Reply:
This doesn't necessarily need to be a gov param and the actual value should be just fine for most of the cases (we could even inline it).

I really don't see an easier way to handle this edge case. Which is, just taking into account the nature of transactions submission that may not be committed at the height of the submission (for reasons like mempool congestion, network latency...)